packages/mobile/src/screens/product-info/components/seller-info/seller-info.tsx

```diff
@@ -52,7 +52,7 @@ const SellerInfo = ({ author }: { author: UserProfileDto }) => {
                 globalStyles.fs14,
               ]}
             >
-              {firstName}
+              {firstName} {lastName}
             </Text>
```

Comment:
may be better use `{'${firstName} ${lastName}'}`  that we will have one component, not three?